custom/icds_reports/tests/reports/test_awc_reports.py

```diff
@@ -2544,24 +2544,24 @@ def test_pregnant_details_first_record_second_trimester(self):
                 data['data'][1],
                 [
                     {'opened_on': datetime.date(2017, 5, 12),
-                     'tt_taken': u'N',
```

Comment:
@Rohit25negi Heads up, files that contain ```from __future__ import unicode_literals``` don't need to include the unicode ```u``` prefix.  It just clutters the code, and shouldn't be relied upon to distinguish between unicode and bytes, since all un-prefixed string literals are unicode by default in python 3.